src/pages/404.js

```diff
@@ -7,28 +7,75 @@ import styled from 'styled-components'
 import remcalc from 'remcalc'
 import ReactMarkdown from 'react-markdown'
 
+import BlueBackground from '../components/Common/BlueBackground'
 import { SectionTitle, BodyPrimary } from '../components/Typography'
-import Button from '../components/Common/Button'
 import Layout from '../components/layout'
 
-const HomePageLink = styled(Button)`
-  width: ${remcalc(145)};
-  display: flex;
-  justify-content: center;
+import { LogoStyleContext } from '../context/PageContext'
+
+import illustration from '../images/404-illustration.svg'
+
+// Layout
+
+const StyledBlueBackground = styled(BlueBackground)`
+  margin-top: -${remcalc(36)};
 `
 
-const StyledBodyPrimary = styled(BodyPrimary)`
-  padding-bottom: ${({ theme }) => theme.space[6]};
+const StyledRow = styled(Row)`
+  align-items: center;
 `
 
-const StyledCol = styled(Col)`
-  margin: ${({ theme }) => theme.space[5]} 0;
+const TextCol = styled(Col).attrs({
+  width: [1, 1, 1, 1, 0.5, 0.5]
+})`
+  padding-top: ${({ theme }) => theme.space[4]};
+  ${breakpoint('smallTablet')`
+    padding-bottom: ${({ theme }) => theme.space[4]};
+  `}
+  ${breakpoint('tablet')`
+    padding-top: ${({ theme }) => theme.space[7]};
+    padding-bottom: ${({ theme }) => theme.space[8]};
+  `}
+`
 
+const IllustrationCol = styled(Col).attrs({
+  width: [1, 1, 1, 1, 0.5, 0.5]
+})`
+  ${breakpoint('smallTablet')`
+    padding-top: ${({ theme }) => theme.space[4]};
+    padding-bottom: ${({ theme }) => theme.space[4]};
+  `}
   ${breakpoint('tablet')`
-    margin: ${({ theme }) => theme.space[8]} 0
+    padding-top: unset;
+    padding-bottom: unset;
   `}
 `
 
+// Paragraphs / Texts
+
+const TitleHeadline = props => (
+  <SectionTitle {...props} as="h1" reverse="true" />
+)
+
+const CopyText = props => <BodyPrimary {...props} reverse="true" />
+
+const LinksTitle = styled(BodyPrimary).attrs({
+  reverse: 'true',
+  bold: 'true'
+})`
+  padding-top: ${({ theme }) => theme.space[3]};
+`
+
+const LinkParagraph = styled('p')`
```

Comment:
We have all of the typography in styled components [here](https://github.com/yldio/yld.io/tree/master/src/components/Typography), have a look at storybook to get a feel of which one you need to use here.

As a rule of thumb on the project we pretty much always use predefined typography components

Reply:
The only one I can find that would be about right is `BodyPrimary`, but I have to undo the padding and set reverse again (instead of relying on the color from the blue background). Pushed it but not sure if that's worth it?
It seems that `BodyPrimary` is more for paragraphs in proper articles, not sure if there's a more plain styled `<p>`?

Reply:
It's nested quite deeply but they have modifiers, `noPadding` , `noPaddingTop` etc. 
Then you can pass reverse as prop also. 

https://github.com/yldio/yld.io/blob/blog-add-mdx/src/components/Typography/modifiers.js#L4

Reply:
Cool, using them now